stdlib/Profile/src/Allocs.jl

```diff
@@ -156,7 +160,7 @@ end
 # Without this, the Alloc's stacktrace prints for lines and lines and lines...
 function Base.show(io::IO, a::Alloc)
     stacktrace_sample = length(a.stacktrace) >= 1 ? "$(a.stacktrace[1]), ..." : ""
-    print(io, "$Alloc($(a.type), $StackFrame[$stacktrace_sample], $(a.size))")
+    print(io, "$Alloc($(a.type), $StackFrame[$stacktrace_sample], $(a.size), $(a.task), $(a.timestamp))")
```

Comment:
Why show them here? I don't think we do that for the sampling profiler?

Reply:
Yeah, no reason, removed